sympy/solvers/ode.py

```diff
@@ -3964,6 +4010,24 @@ def ode_2nd_power_series_regular(eq, func, order, match):
             return Eq(f(x), collect(finalseries1 + finalseries2,
                 [C0, C1]) + Order(x**terms))
 
+def ode_2nd_linear_bessel(eq, func, order, match):
+    r"""
+    Gives solution of the Bessel differential equation
+    .. math :: x**2*\frac{d^2y}{dx^2} + x*\frac{dy}{dx}*y(x) + (x**2-n**2)*y(x)
+       if n is integer then the solution is of the form Eq(f(x), C0*besselj(n,x) + C1*bessely(n,x)) as both the solutions are linearly independant
+       else if n is a fraction then the solution is of the form Eq(f(x), C0*besselj(n,x) + C1*besselj(-n,x)) which can also transform into Eq(f(x), C0*besselj(n,x) + C1*bessely(n,x)).
```

Comment:
These lines are too long.

Can you format the docstring like e.g. here:
[https://github.com/sympy/sympy/blob/1b7514db3923012f6f0aad14315b24e95bef639c/sympy/solvers/ode.py#L3580](https://github.com/sympy/sympy/blob/1b7514db3923012f6f0aad14315b24e95bef639c/sympy/solvers/ode.py#L3580)

Reply:
okkk

Reply:
done